### ISSUE TITLE
[FR] Only apply image resolution scale to the dock preview, use unscaled for large preview

### ISSUE THREAD

**Issue by bsiegel** [ejbills/DockDoor#276]
## Problem or Need
I use a very small dock preview size, so I use image resolution scale 4 in order to achieve best performance from this tool. However when I long-hover the dock preview, I want to see a full-resolution large preview - even when the tool is configured with a window resolution scale of 1 it still results in some blurriness in the large preview.

## Proposed Solution
It would be nice if the image resolution scale value only affected the dock preview, and the large preview always used full resolution (unscaled).

## Alternative Solutions
Use lowest image resolution scale. Though there's no current way to see an unscaled large preview.

## Potential Impact
It would look nicer and more polished, and also allow users who choose small dock preview size to achieve best performance without compromising the quality of the large preview.

## macOS Version
14.6.1

## App Version
1.1.5

- [x] I have reviewed the open issues to ensure this is not a duplicate feature request

**Comment by ejbills** [comment]
This wouldn't work for windows in other spaces, minimized windows, hidden windows, etc. It is quite crucial that the expanded window is the same image as the mini preview. Unfortunately, I don't think this can be added at this time.

**Comment by illavoluntas** [comment]
Yes, you would have to capture the windows 2 times... It's quite redundant and resource-consuming.
